### ISSUE TITLE
Craft 4 Support

### ISSUE THREAD

**Issue by internetztube** [amici-infotech/craft-super-dynamic-fields#3]
First of all, thank you for this plugin. Now that Craft 4 is about to be released, do you plan to have the plugin ready by then?

**Comment by mufi-siebird** [comment]
Of course. Plugin will be available in week or 2 for craft 4.

**Comment by amici-infotech** [comment]
Of course. Plugin will be available in week or 2 for craft 4.

**Comment by internetztube** [comment]
Thank you!

**Comment by internetztube** [comment]
Do you have a rough timeline for when the first beta version of the plugin will be available? @amici-infotech

**Comment by amici-infotech** [comment]
Well, Alpha version is ready to work with. 
https://plugins.craftcms.com/super-dynamic-fields?cmsConstraint=%5E4.0

You can use it by adding this in your composer.json file:
```
"minimum-stability": "dev",
"prefer-stable": true,
```

Then use this command:
```
composer require amici/craft-super-dynamic-fields:v2.0.0-alpha.1 --dev
```

**Comment by amici-infotech** [comment]
We are doing some Testing in here. It will be available in 2 weeks as `v2.0.0`

**Comment by internetztube** [comment]
Awesome! Thank you!
